### PR TITLE
Migrate more pallet tests to construct_runtime 

### PR DESCRIPTION
**Pull request by ascjones**
Part of https://github.com/paritytech/substrate/issues/7949. Migrates the following crate **test** runtimes to `construct_runtime` :

- bounties
- contracts
- democracy

### CODE REVIEW

File: frame/bounties/src/tests.rs [hunk 1]
```diff
@@ -142,10 +139,8 @@ impl Config for Test {
 	type MaximumReasonLength = MaximumReasonLength;
 	type WeightInfo = ();
 }
-type System = frame_system::Module<Test>;
-type Balances = pallet_balances::Module<Test>;
-type Treasury = pallet_treasury::Module<Test>;
-type Bounties = Module<Test>;
+
+type TreasuryEvent = pallet_treasury::Error::<Test, pallet_treasury::DefaultInstance>;
```
Comment (thiolliere) [inline frame/bounties/src/tests.rs:143]: This should be renamed TreasuryError

**Review by shawntabrizi — APPROVED** [review]
If this compiles, looks good to me

**Review by thiolliere — CHANGES_REQUESTED** [review]

File: frame/bounties/src/tests.rs (hunk 1 above)
Comment (ascjones) [reply frame/bounties/src/tests.rs:143]: Er yeah just checking you were paying attention :sweat_smile:

**Review by thiolliere — APPROVED** [review]

**Comment by thiolliere** [comment]
bot merge

**Pull request merged**